i18n-build/src/gettext_impl/mod.rs

```diff
@@ -407,11 +407,13 @@ pub fn run(crt: &Crate) -> Result<()> {
                 .subcrates
                 .iter()
                 .map(|subcrate_path| {
-                    Crate::from(
-                        subcrate_path.clone(),
-                        Some(crt),
-                        crt.config_file_path.clone(),
-                    )
+                    let subcrate_config_path =
```

Comment:
This changes the behaviour from using the parent's custom config path to using the child's custom config path, then falling back to the basic `i18n.toml`. I'm not sure in what cases the old behaviour would have applied, so I'd appreciate your input on whether this is the best way to go about this change.